seacrowd/sea_datasets/palito/palito.py

```diff
@@ -0,0 +1,167 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (DEFAULT_SEACROWD_VIEW_NAME,
+                                      DEFAULT_SOURCE_VIEW_NAME, Tasks)
+
+_DATASETNAME = "palito"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
+
+_CITATION = """
+@inproceedings{dita-etal-2009-building,
+    title = "Building Online Corpora of {P}hilippine Languages",
+    author = "Dita, Shirley N.  and
+      Roxas, Rachel Edita O.  and
+      Inventado, Paul",
+    editor = "Kwong, Olivia",
+    booktitle = "Proceedings of the 23rd Pacific Asia Conference on Language, Information and Computation, Volume 2",
+    month = dec,
+    year = "2009",
+    address = "Hong Kong",
+    publisher = "City University of Hong Kong",
+    url = "https://aclanthology.org/Y09-2024",
+    pages = "646--653",
+}
+"""
+
+# We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LANGUAGES = {
+    "bik": "Bikol",
+    "ceb": "Cebuano",
+    "hil": "Hiligaynon",
+    "ilo": "Ilocano",
+    "tgl": "Tagalog",
+    "pam": "Kapampangan",
+    "pag": "Pangasinense",
+    "war": "Waray",
+}
+
+_LOCAL = False
+
+_DESCRIPTION = """\
+This paper aims at describing the building of the online corpora on Philippine
+languages as part of the online repository system called Palito. There are five components
+of the corpora: the top four major Philippine languages which are Tagalog, Cebuano,
+Ilocano and Hiligaynon and the Filipino Sign Language (FSL). The four languages are
+composed of 250,000-word written texts each, whereas the FSL is composed of seven
+thousand signs in video format. Categories of the written texts include creative writing (such
+as novels and stories) and religious texts (such as the Bible). Automated tools are provided
+for language analysis such as word count, collocates, and others. This is part of a bigger
+corpora building project for Philippine languages that would consider text, speech and
+video forms, and the corresponding development of automated tools for language analysis
+of these various forms.
+"""
+
+_HOMEPAGE = "https://github.com/imperialite/Philippine-Languages-Online-Corpora/tree/master/PALITO%20Corpus"
+
+_LICENSE = "GNU Lesser General Public License family (lgpl)"
```

Comment:
use `_LICENSE` from this:

```python
from seacrowd.utils.constants import Licenses
```

```suggestion
_LICENSE = Licenses.LGPL.value
```

Reply:
Updated 👍